src/main/java/com/google/identitytoolkit/GitkitClient.java

```diff
@@ -446,7 +446,7 @@ private JSONObject buildChangeEmailRequest(HttpServletRequest req, String gitkit
     private final String recipient;
 
     public OobResponse(String responseBody) {
-      this(null, null, Optional.<String>absent(), null, ERROR_PREFIX + responseBody + " }");
+      this(null, null, Optional.<String>absent(), null, ERROR_PREFIX + "\"" +  responseBody + "\" }");
```

Comment:
Could you append the first "\"" to ERROR_PREFIX directly at line 440, like this
ERROR_PREFIX = "{\"error\": \"";